back/config/schemas/settings.schema.json.erb

```diff
@@ -502,10 +502,18 @@
             "description": "The boundary of the municipality/organisation is specified by its Relation ID in OpenStreetMap (OSM). You can find the Relation ID at https://www.openstreetmap.org. Search for the place, click on the correct result and you'll find the Relation ID if the result is a relation (e.g. \"Relation: Knokke-Heist (4569)\"). If there is no relation defined for the place (e.g. Scheveningen), search for the next largest administrative territory (e.g. The Hague)."
           },
           "esri_integration": {
-            "type": "boolean",
-            "default": false,
             "title": "Esri Integration",
-            "description": "Enable Esri integration for maps. In development. DO NOT ENABLE ON COMMERCIAL PLATFORM!"
+            "description": "Esri integration for maps. In development. DO NOT ENABLE ON COMMERCIAL PLATFORM!",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "enabled": { "type": "boolean", "default": false},
+              "api_key": {
+                "title": "API Key",
+                "description": "The API key for the Esri integration. (TODO: add more useful information)",
+                "type": "string"
```

Comment:
Should it have `"private": true`? See for details https://github.com/CitizenLabDotCo/citizenlab/blob/650ae6dc85e9edd859ef6514a884f8080714a0e3/back/app/services/settings_service.rb#L70-L78

Reply:
Yes, it probably should! Thanks!

Reply:
(just in case, it should not, if FE needs it)

Reply:
Sorry, what do you mean by "(just in case, it should not, if FE needs it)"?

Reply:
It should not be `private` if FE needs to access it. If it'll be used only on BE, it should be private.

Reply:
Hmm, I think FE needs it, but maybe we can/should find a way to avoid the FE needing it. A point to be refined.